src/zenml/config/docker_settings.py

```diff
@@ -171,33 +160,32 @@ class DockerSettings(BaseSettings):
         environment: Dictionary of environment variables to set inside the
             Docker image.
         build_config: Configuration for the main image build.
-        dockerignore: DEPRECATED, use build_config.dockerignore instead.
-        copy_files: DEPRECATED, use the `source_files` attribute instead.
-        copy_global_config: DEPRECATED/UNUSED.
         user: If not `None`, will set the user, make it owner of the `/app`
             directory which contains all the user code and run the container
             entrypoint as this user.
-        source_files: Defines how the user source files will be handled when
-            building the Docker image.
-            * INCLUDE: The files will be included in the Docker image.
-            * DOWNLOAD: The files will be downloaded when running the image. If
-              this is specified, the files must be inside a registered code
-              repository and the repository must have no local changes,
-              otherwise the build will fail.
-            * DOWNLOAD_OR_INCLUDE: The files will be downloaded if they're
-              inside a registered code repository and the repository has no
-              local changes, otherwise they will be included in the image.
-            * IGNORE: The files will not be included or downloaded in the image.
-              If you use this option, you're responsible that all the files
-              to run your steps exist in the right place.
+        allow_including_files_in_images: If `True`, code can be included in the
+            Docker images if code download from a code repository or artifact
+            store is disabled or not possible.
+        allow_download_from_code_repository: If `True`, code can be downloaded
+            from a code repository if possible.
+        allow_download_from_artifact_store: If `True`, code can be downloaded
+            from the artifact store.
+        build_options: DEPRECATED, use parent_image_build_config.build_options
+            instead.
+        dockerignore: DEPRECATED, use build_config.dockerignore instead.
+        copy_files: DEPRECATED/UNUSED.
```

Comment:
Attributes like these were deprecated before and they were pointing to the `source_files` which is deprecated now. Perhaps, it might be a nice time to remove some of these.